bin/yaml/solidity.yaml

```diff
@@ -11,11 +11,32 @@ compilers:
       - 0.6.12
       - 0.7.6
       - 0.8.25
-  solidity_eravm:
```

Comment:
Hi! Can you give me a bit of insifght into the naming change? Thanks!

Reply:
hey @RubenRBS !

Sure, let me explain a bit, because this is not completely clear from the yaml.

The initial `solidity` compiler in this file is the main `solc` compiler supported by the Ethereum folks to compile Solidity into Ethereum Virtual Machine (EVM). It's left untouched.

In ZKsync, we developed our own compiler that is called `zksolc` (added at the end of the file) that can compiler Solidity into EraVM (ZKsync-specific L2 virtual machine), and also EVM.

The tricky thing here is that the ZKsync compiler `zksolc` depends on ZKsync custom fork of the `solc` compiler that is added here as `solidity_zksync`. This custom fork is not included in the final interface, but used only as a dependency for the `zksolc` compiler (see `options` [here](https://github.com/compiler-explorer/compiler-explorer/pull/6592/files#diff-1a06a1796eb79f455a8124c935741be7878f57998fa700f34460ca8b9054a3d3R38))

I hope this answers the renaming thing.